apollo-router/src/plugins/telemetry/metrics/apollo/mod.rs

```diff
@@ -187,7 +188,10 @@ impl Config {
             )),
             Box::new(
                 CustomAggregationSelector::builder()
-                    .boundaries(default_buckets())
+                    .boundaries(
+                        // Returns [~1.4ms ... ~5min]
```

Comment:
I think we could add some additional comments about how this maps to the "Apollo" histogram format here.

> a duration x in μs is counted in the bucket of index max(0, min(ceil(ln(x)/ln(1.1)), 383)).